umbraco-cloud/set-up/project-settings/README.md

```diff
@@ -120,6 +122,21 @@ On your Umbraco Cloud project, it is possible to see the usage of Custom Domains
 
 You can see metrics related to the overall health and performance of the Azure app service hosting the live environment of your solution.
 
+## Automatic Upgrades
```

Comment:
I also think it makes sense to link the the page that explains how to enable this feature 😄

In that case it is this page:  https://docs.umbraco.com/umbraco-cloud/product-upgrades/minor-upgrades

Reply:
link added